wagtail/admin/templates/wagtailadmin/permissions/includes/collection_member_permissions_formset.html

```diff
@@ -17,9 +17,9 @@
     <table class="listing">
         <col />
         {% for i in formset.permission_types %}
-            <col width="15%" />
```

Comment:
We have 6 permission types. Using `15%` would mean the columns take up `90%` of the table width in total. Then the remaining `10%` is shared for the first (page title) and last (delete button) columns.

Change this to use `5%` so the checkboxes take up `30%`, then change the delete button to take up `10%`, which means the remaining 60% would be for the page title.